### PR TITLE
taxonomy_api.md: About identifier+taxonomy uniqueness

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      | 4.5+
| Edition       | Headless+

Insired by https://github.com/ibexa/documentation-developer/pull/2212#discussion_r1383842516

Document taxonomy entry identifier

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/content_management/taxonomy/taxonomy_api.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ and use `TaxonomyServiceInterface::loadEntryByIdentifier()`:
 [[= include_file('code_samples/api/public_php_api/src/Command/TaxonomyCommand.php', 43, 46) =]]
 ```
 
+!!! note
+
+    A taxonomy entry identifier is unique per taxonomy. If you have [several taxonomies](taxonomy.md#customize-taxonomy-structure), you can increase code readability by always pass the taxonomy identifier event when it is the default one.
```
Comment (ms2ad) [inline docs/content_management/taxonomy/taxonomy_api.md:22]:
```suggestion
    A taxonomy entry identifier is unique per taxonomy. If you have [several taxonomies](taxonomy.md#customize-taxonomy-structure), you can increase code readability by always pass the taxonomy identifier even when it is the default one.
```

File: docs/content_management/taxonomy/taxonomy_api.md [hunk 2]
```diff
@@ -17,6 +17,15 @@ and use `TaxonomyServiceInterface::loadEntryByIdentifier()`:
 [[= include_file('code_samples/api/public_php_api/src/Command/TaxonomyCommand.php', 43, 46) =]]
 ```
 
+!!! note
+
+    A taxonomy entry identifier is unique per taxonomy. If you have [several taxonomies](taxonomy.md#customize-taxonomy-structure), you can increase code readability by always pass the taxonomy identifier even when it is the default one.
```
Comment (adriendupuis) [inline docs/content_management/taxonomy/taxonomy_api.md:22]:
```suggestion
    A taxonomy entry identifier is unique per taxonomy. If you have [several taxonomies](taxonomy.md#customize-taxonomy-structure), you can increase code readability by always passing the taxonomy identifier even when it's the default one.
```

**Review by juskora — APPROVED** [review]

File: docs/content_management/taxonomy/taxonomy_api.md [hunk 3]
```diff
@@ -17,6 +17,15 @@ and use `TaxonomyServiceInterface::loadEntryByIdentifier()`:
 [[= include_file('code_samples/api/public_php_api/src/Command/TaxonomyCommand.php', 43, 46) =]]
 ```
 
+!!! note
+
+    A taxonomy entry identifier is unique per taxonomy. If you have [several taxonomies](taxonomy.md#customize-taxonomy-structure), you can increase code readability by always passing the taxonomy identifier even when it's the default one.
```
Comment (ms2ad) [inline docs/content_management/taxonomy/taxonomy_api.md:22]: It should be specified what the default taxonomy is or better reference the method returning the default taxonomy.

**Review by ms2ad — CHANGES_REQUESTED** [review]

**Comment by ms2ad** [comment]
I would have liked in the relevant release notes to also have a note when the uniqueness changed from globally to per taxonomy.

File: docs/content_management/taxonomy/taxonomy_api.md (hunk 3 above)
Comment (adriendupuis) [reply docs/content_management/taxonomy/taxonomy_api.md:22]: @ms2ad
```suggestion
    A taxonomy entry identifier is unique per taxonomy. If you have [several taxonomies](taxonomy.md#customize-taxonomy-structure), you can increase code readability by always passing the taxonomy identifier even when it's the default one. The default taxonomy is `tags` if it exists, else the first configured taxonomy (see `\Ibexa\Taxonomy\Service\TaxonomyConfiguration::getDefaultTaxonomyName` for details).
```

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 4]
```diff
@@ -195,6 +195,11 @@ Product search now supports aggregations, with the following aggregations availa
 
 The new [TaxonomyEntryIdAggregation](https://doc.ibexa.co/en/4.5/search/aggregation_reference/taxonomyentryid_aggregation/) aggregates results based on content taxonomy entries or product categories.
 
+### Tag identifiers
+
+The taxonomy entry identifier uniqueness as been changed from globally unique to unique per taxonomy.
+It's no longer necessary to take other taxonomies into account when creating tags in a taxonomy.
+
```
Comment (adriendupuis) [inline docs/release_notes/ibexa_dxp_v4.5.md:202]: @adamwojs Does it really worth being mentioned?
@juskora If it worths it, please, check style.

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 5]
```diff
@@ -195,6 +195,11 @@ Product search now supports aggregations, with the following aggregations availa
 
 The new [TaxonomyEntryIdAggregation](https://doc.ibexa.co/en/4.5/search/aggregation_reference/taxonomyentryid_aggregation/) aggregates results based on content taxonomy entries or product categories.
 
+### Tag identifiers
+
+The taxonomy entry identifier uniqueness as been changed from globally unique to unique per taxonomy.
```
Comment (juskora) [inline docs/release_notes/ibexa_dxp_v4.5.md:200]:
```suggestion
The taxonomy entry identifier uniqueness has been changed from globally unique to unique per taxonomy.
```

**Review by juskora — APPROVED** [review]

**Pull request merged**